### ISSUE TITLE
Failed to open users list file! (rfid_users_list.db)

### ISSUE THREAD

**Issue by AshrafMikhael** [HausnerR/esphome-door-lock#7]
Hello 
Thanks for your great esphome-door-look.
Everything is working fine, except that error of opening users list file and I receive the below error.
Where should I put this file.
 
[0;35m[C][switch.gpio:011]: Setting up GPIO Switch 'Door lock'...[0m
[0;36m[D][switch:017]: 'Door lock' Turning OFF.[0m
[0;36m[D][switch:037]: 'Door lock': Sending state OFF[0m
[0;36m[D][switch:017]: 'Door lock' Turning OFF.[0m
[0;36m[D][rfid:062]: Loading users list from file...[0m
[1;31m[E][rfid:066]: Failed to open users list file![0m

Thanks 
Ashraf

**Comment by BartEngelen** [comment]
Did you send the users list with Home Assistant service Upload Users?

**Comment by AshrafMikhael** [comment]
Yes, and I got error

[0;36m[D][rfid:100]: Saving users list to file...[0m
[1;31m[E][rfid:104]: Failed to open users list file![0m

I think the error due to the users list file "rfid_users_list.db" is not uploaded to esp8266.
how can i upload this file?
Thanks

**Comment by BartEngelen** [comment]
That's strange did you use the following structure to upload the users: (Services: RFID_UPLOAD_USERS_LIST)
[
"XX:XX:XX:XX=Example 1",
"XX:XX:XX:XX=Example 2"
]

**Comment by AshrafMikhael** [comment]
Sure, I use this structure. I call the service from HA Service "ESPHome: door_lock_upload_users_list"

I am using HA ESPhome to compile and install the code on ESP8266 MCU.

I uploaded the file "rfid_users_list.db" to HA under "/config/esphome" folder, is this the correct location for this file?

Also I tried to upload it directly to esp8266 using arduino IDE DATA Upload tool but no clue.

Thanks
